package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-mi-humidifier",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
```suggestion
  "version": "2.1.0",
```

Reply:
I will bump version when PR will be merged in master. I usually use `npm version` command for this because it also creates matching git tag